package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@emotion/react": "^11.10.5",
     "@emotion/styled": "^11.10.5",
+    "@material-ui/core": "^4.12.4",
```

Comment:
I think the `material-ui/core` is an older version of material UI. You should use `@mui/material/` like the other files do. Should remove this dependency.

Reply:
fixed